hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/ITestAbfsInputStreamReadFooter.java

```diff
@@ -71,22 +94,46 @@ public void testMultipleServerCallsAreMadeWhenTheConfIsFalse()
   private void testNumBackendCalls(boolean optimizeFooterRead)
       throws Exception {
     int fileIdx = 0;
+    final List<Future> futureList = new ArrayList<>();
+    for (int i = 0; i <= 4; i++) {
+      final int fileSize = (int) Math.pow(2, i) * SIZE_256_KB;
+      final int fileId = fileIdx++;
+      Future future = executorService.submit(() -> {
+        try {
+          try (AzureBlobFileSystem spiedFs = createSpiedFs(
```

Comment:
Why do we need a try inside try?
If we are catching a general exception, can we have a single exception?

Reply:
Fixed it.